src/classes/api/lumedicApi.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+export default {
+    postData: (values) =>
+    {
+    console.log(values);
+    axios.post('https://api-dev.lumedic.id/portal/v1/patient/validate', values).then(res => {
```

Comment:
do not use .then() here, instead, return the data to the component from where this API method is called